docs/ibc/overview.md

```diff
@@ -6,170 +6,205 @@ parent:
-contain a sequence to optionally enforce ordering. IBC packets also contain a `TimeoutTimestamp` and
-`TimeoutHeight`, which when non-zero, will determine the deadline before which the receiving module
-must process a packet. If the timeout passes without the packet being successfully received, the
-sending module can timeout the packet and take appropriate actions.
+`channelID`. This packet structure allows modules to know the sender module of a given packet. IBC packets 
+contain a sequence to optionally enforce ordering. 
+
+IBC packets also contain a `TimeoutHeight` and a `TimeoutTimestamp` that determine the deadline before the receiving module must process a packet. 
 
 Modules send custom application data to each other inside the `Data []byte` field of the IBC packet.
-Thus, packet data is completely opaque to IBC handlers. It is incumbent on a sender module to encode
-their application-specific packet information into the `Data` field of packets, and the receiver
-module to decode that `Data` back to the original application data.
+Thus, packet data is opaque to IBC handlers. It is incumbent on a sender module to encode
+their application-specific packet information into the `Data` field of packets. The receiver
+module must decode that `Data` back to the original application data.
 
 ### [Receipts and Timeouts](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel)
 
 Since IBC works over a distributed network and relies on potentially faulty relayers to relay messages between ledgers, 
-IBC must handle the case where a packet does not get sent to its destination in a timely manner or at all. Thus, packets must 
-specify a timeout height or timeout timestamp after which a packet can no longer be successfully received on the destination chain.
+IBC must handle the case where a packet does not get sent to its destination in a timely manner or at all. Packets must 
+specify a non-zero value for timeout height (`TimeoutHeight`) or timeout timestamp (`TimeoutTimestamp` ) after which a packet can no longer be successfully received on the destination chain.
+
+- The `timeoutHeight` indicates a consensus height on the destination chain after which the packet is no longer be processed, and instead counts as having timed-out.
+- The `timeoutTimestamp` indicates a timestamp on the destination chain after which the packet is no longer be processed, and instead counts as having timed-out.
+
+If the timeout passes without the packet being successfully received, the
+sending module can timeout the packet and take appropriate actions.
+
+If the timeout is reached, then a proof of packet timeout can be submitted to the original chain. The original chain can then perform 
+application-specific logic to timeout the packet, perhaps by rolling back the packet send changes (refunding senders any locked funds, etc.).
+
+- In ORDERED channels, a timeout of a single packet in the channel causes the channel to close. 
```

Comment:
since line 175 defines what happens for a packet timeout in an ORDERED channel, I attempted to describe what happens in an UNORDERED channel  on line 180

(we really gotta show this essential content some love... am I on the right track here?)